mkdocs.yml

```diff
@@ -27,6 +27,10 @@ markdown_extensions:
     - pymdownx.snippets
     - toc:
         permalink: true
+    - attr_list
+    - pymdownx.emoji:
+        emoji_index: !!python/name:materialx.emoji.twemoji
+        emoji_generator: !!python/name:materialx.emoji.to_svg
```

Comment:
Needed because of https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#configuration